ai-service/src/ai_service/handlers/ingest.py

```
@@ -58,7 +62,7 @@
             embeddings = embed_documents(code_snippets)
 
             add_chunks(code_snippets, embeddings)
-            logger.info(f"Stored {len(code_snippets)} code snippets in ChromaDB.")
+            logger.info(f"Stored {len(code_snippets)} code chunks in ChromaDB.")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)